watertap/property_models/anaerobic_digestion/tests/test_adm1_reaction.py

```diff
@@ -105,8 +106,8 @@ def test_build(self, model):
             ("R1", "Liq", "X_pr"): 0.20,
             ("R1", "Liq", "X_li"): 0.30,
             ("R1", "Liq", "X_I"): 0.2,
-            ("R1", "Liq", "S_IC"): 1.46907e-5 * mw_c,
-            ("R1", "Liq", "S_IN"): -1e-18 * mw_n,
+            ("R1", "Liq", "S_IC"): -8e-19 * mw_c,
```

Comment:
Some of these are substantially different. Why, and is this expected?

Reply:
It was changed with respect to the changes made in the reaction stochiometric dictionary, getting rid of some assumptions we made